distribution/docs/src/main/resources/content/_configuring/configuring-from-import-intro-contents.adoc

```diff
@@ -1,14 +1,32 @@
-:title: Configuration Export/Import Intro
+:title: Exporting/Importing Configuration Settings
 :type: configuringImport
 :status: published
 :summary: Export a current ${branding} configuration and use it to restore the system to the same state on a brand new installation.
 :order: 00
 
-The Configuration Export/Import capability allows administrators to export the current ${branding} configuration and use it to restore the same state on a brand new installation or when adding a second node to a highly available deployment.
+==== Exporting/Importing Configuration Settings
+
+The Configuration Export/Import capability allows administrators to export the current ${branding} configuration and use it to restore the same state for either a brand new installation or a second node for a highly available deployment.
+
+To export the current configuration settings:
+
+. Run the `migration:export` command from the ${command-console}.
+. A file named `exported-${project.version}.zip` will be created in the `exported` directory underneath `<${branding}_HOME>`, but an alternative location can be specified with the `migration:export`. Copy this file to a secure place.
+
+To import previously exported configuration settings:
+
+. Step through the installation process.
+. Launch the newly installed ${branding}.
+. Copy the previously exported file from your secure place to the `exported` directory underneath `<${branding}_HOME>`, but an alternative location can be specified with the `migration:import`.
```

Comment:
Should we note that the exported directory will need to be created?  It is not there by default...right?